src/test/rgw/test_rgw_aws.cc

```diff
@@ -0,0 +1,64 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+#include <gtest/gtest.h>
+#include <aws/core/Aws.h>
+#include <rgw/rgw_aws.h>
+#include <chrono>
+#include <thread>
+#include "common/ceph_context.h"
+#include <aws/core/auth/AWSCredentialsProvider.h>
+/*
+for running the unit tests, please install moto_server using
+pip install moto
+See http://docs.getmoto.org/en/latest/docs/server_mode.html for  more details.
+Note: lambda mock server requires you to setup docker.
+run the server using
+> moto_server lambda -p5555
+> moto_server sns -p4444
+
+ Now create sns topic, and lambda function in moto_server using
+  aws sns --endpoint-url=http://localhost:4444 create-topic --name gsoc20-ceph
+  aws --endpoint-url=http://localhost:5555 lambda create-function --function-name gsoc20 --zip-file fileb://<file.zip> --handler index.handler --runtime nodejs12.x --role lambda-ex
+
+ Replace the ARN obtained in the function below
```

Comment:
can we write a sample command that will give the ARN we expect?
even a manual test should not require you to recompile the code :-)